### DISCUSSION TITLE
Call Component class from a html file

### DISCUSSION THREAD

**Discussion by enderahmetyurt** [yippee-fun/phlex#28] [Help] [answered]
Hi,

I created `NavComponent` like README. I callled in my html file something like

`<%= render(NavComponent.new.template) %>`

I am getting this error

```
uninitialized constant #<Class:0x0000000109739f88>::NavComponent

            @virtual_path = "articles/index";;@output_buffer.append=( render(NavComponent.new.template) );
```

How should I call my component from a html file?

Cheers.

**Comment by joeldrapper** [comment, marked as answer]
You should be able to include Phlex components like this.

```erb
<%= NavComponent.new.call.html_safe %>
```

But it looks like your `NavComponent` hasn't even been loaded. If you put the `NavComponent` in `app/components/nav_component.rb` it should be loaded automatically. I’d like to create a helper method for Rails so you can include them with just `<%= component NavComponent %>`. This should be part of #1.

**Reply by enderahmetyurt** [reply]
Actually I did these things before, but I forgot to restart `rails s` :) It works after restarting the server.
